includes/Emails/VendorWithdrawRequest.php

```
@@ -93,8 +94,9 @@
         $this->setup_locale();
         $this->object                          = $seller;
         $this->placeholders['{store_name}']    = $seller->get_shop_name();
-        $this->placeholders['{amount}']        = dokan()->email->currency_symbol( $amount );
-        $this->placeholders['{charge}']        = dokan()->email->currency_symbol( $withdraw->get_charge() );
+        // Format amounts using wc_price() so WooCommerce "Number of decimals" setting is respected
+        $this->placeholders['{amount}']        = dokan()->email->currency_symbol( wc_format_decimal( $amount, $decimals ,true ) );
+        $this->placeholders['{charge}']        = dokan()->email->currency_symbol( wc_format_decimal( $withdraw_charger, $decimals ,true ) );
```

PHPMD (2.15.0)

Avoid unused local variables such as '$withdraw_charger'. (undefined)

(UnusedLocalVariable)